src/App.tsx

```diff
@@ -1,9 +1,37 @@
+import { Routes, Route } from 'react-router-dom';
+
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { HomePage } from './pages/HomePages';
+import { Phones } from './pages/Phones';
+import { Tablets } from './pages/Tablets';
+import { Accessories } from './pages/Accessories';
+import { Favourites } from './pages/Favourites';
+import { Cart } from './pages/Cart';
+import { ProductDetails } from './pages/ProductDetails';
+
 import './App.scss';
 
-const App = () => (
-  <div className="App">
-    <h1>React Phone Catalog</h1>
-  </div>
-);
+const App = () => {
+  return (
+    <div className="App">
+      <Header />
+      <main>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/phones" element={<Phones />} />
+          <Route path="/phones/:productId" element={<ProductDetails />} />
```

Comment:
could be rewritten to
```suggestion
          <Route path="/phones" element={<Phones />}>
            <Route path=":productId" element={<ProductDetails />} />
          </Route>
```